zypper-formula/CHANGELOG.rst

```diff
@@ -0,0 +1,9 @@
+grains formula
```

Comment:
s/grains/zypper/

Reply:
Thanks, I'll go through the files and do some clean ups before finalizing the import.

Reply:
LICENSE probably also needs an update ;-)

Reply:
Hm, do you think the copyright holder should be changed? I figured it'd be best to keep the original one.

Reply:
I added a second entry with the recent years, I think that should be good.